### ISSUE TITLE
Input component Errors out on React Native: Cannot read property useRef of null

### ISSUE THREAD

**Issue by ramspeedy** [tamagui/tamagui#2680]
### Current Behavior

Working on bare React Native app, installed tamagui and using v3 config. Hit error when using Input component and running on iOS sim.

### Expected Behavior

Expect no errors while rendering.

### Tamagui Version

```markdown
1.99.1
```

### Platform (Web, iOS, Android)

```markdown
iOS
```

### Reproduction

```markdown
npx react-native@latest init tamagui_test
npm install tamagui
npm install @tamagui/config

Add these imports and create config
import { TamaguiProvider, createTamagui, Input } from 'tamagui';
import { config } from '@tamagui/config/v3'
const tamaguiConfig = createTamagui(config)

Add Input component and try to render. (npm start)
Here is the simple repo I made to test this.
https://github.com/ramspeedy/tamagui-test/tree/main
```

**Comment by ralphhovsepian** [comment]
any solutions found? @ramspeedy

**Comment by ramspeedy** [comment]
Nothing yet, moving forward without the library since I'm on a deadline. Hoping @SamuraiF0x or someone else from the team can comment so I can go back to using it.

**Comment by SamuraiF0x** [comment]
Hi @ramspeedy, I'm adding this to my backlog

**Comment by ralphhovsepian** [comment]
@SamuraiF0x is this issue actual in older versions?

**Comment by ramspeedy** [comment]
> Hi @ramspeedy, I'm adding this to my backlog

Thanks @SamuraiF0x, appreciate the response! I also wanted to echo a sentiment similar to a comment on an issue a few weeks ago. I found the docs hard to follow and a bit unintuitive; I'd be willing to suggest edits and/or rewrite certain sections. If that's of interest to the team, do let me know. I'm on your discord and can chat there.

**Comment by natew** [comment]
@ramspeedy if you send us PRs with improvements I will happily pay for that, DM me `foobully` on discord.

**Comment by SamuraiF0x** [comment]
Hey huys @ralphhovsepian @rk-kc, where do you get this error?

**Comment by rk-kc** [comment]
Hi @SamuraiF0x , when using the Input component. On screen load.

**Comment by SamuraiF0x** [comment]
Could you give me more details please, also native? or expo? android or ios only?

**Comment by rk-kc** [comment]
Expo. Both IOS and android. Tamagui version: ^1.94.5 (Tried the latest version as well 1.99 and same error)

Render Error on Input component load:

`TypeError: Cannot read property 'useRef' of null

This error is located at:
    in Unknown (created by Themed(Anonymous))
    in Theme (created by Themed(Anonymous))
    in Themed(Anonymous) (created by UserProfile)
    in RCTView (created by YStack)
    in YStack (created by UserProfile)
    in RCTView
    in Unknown (created by UserProfile)
    in RCTScrollContentView (created by ScrollView)
    in RCTScrollView (created by ScrollView)
    in ScrollView (created by ScrollView)
    in ScrollView (created by UserProfile)
    in RCTView (created by View)
    in View (created by AnimatedComponent(View))
    in AnimatedComponent(View)
    in Unknown (created by BottomSheetViewComponent)
    in BottomSheetViewComponent (created by UserProfile)
    in RCTView (created by View)
    in View (created by AnimatedComponent(View))
    in AnimatedComponent(View)
    in Unknown (created by PanGestureHandler)
    in PanGestureHandler (created by BottomSheetDraggableViewComponent)
    in BottomSheetDraggableViewComponent (created by BottomSheet)
    in RCTView (created by View)
    in View (created by AnimatedComponent(View))
    in AnimatedComponent(View)
    in Unknown (created by BottomSheet)
    in RCTView (created by View)
    in View (created by AnimatedComponent(View))
    in AnimatedComponent(View)
    in Unknown (created by BottomSheet)
    in RCTView (created by View)
    in View (created by BottomSheetContainerComponent)
    in BottomSheetContainerComponent (created by BottomSheet)
    in BottomSheetGestureHandlersProvider (created by BottomSheet)
    in BottomSheet (created by BottomSheet)
    in TouchableWithoutFeedback (created by UserProfile)
    in RNCSafeAreaView
    in Unknown (created by UserProfile)
    in UserProfile (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by SceneView)
    in RCTView (created by View)
    in View (created by DebugContainer)
    in DebugContainer (created by MaybeNestedStack)
    in MaybeNestedStack (created by SceneView)
    in RCTView (created by View)
    in View (created by SceneView)
    in RNSScreen (created by Animated(Anonymous))
    in Animated(Anonymous) (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by SceneView)
    in SceneView (created by NativeStackViewInner)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by ScreenStack)
    in RNSScreenStack (created by ScreenStack)
    in ScreenStack (created by NativeStackViewInner)
    in NativeStackViewInner (created by NativeStackView)
    in RNCSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by SafeAreaProviderCompat)
    in SafeAreaProviderCompat (created by NativeStackView)
    in NativeStackView (created by NativeStackNavigator)
    in PreventRemoveProvider (created by NavigationContent)
    in NavigationContent
    in Unknown (created by NativeStackNavigator)
    in NativeStackNavigator (created by App)
    in EnsureSingleNavigator
    in BaseNavigationContainer
    in ThemeProvider
    in NavigationContainerInner (created by App)
    in ToastImperativeProvider (created by ToastProvider)
    in Provider (created by ToastProvider)
    in Provider (created by CollectionProvider)
    in CollectionProvider (created by ToastProvider)
    in ToastProvider (created by App)
    in RCTView (created by View)
    in View (created by GestureHandlerRootView)
    in GestureHandlerRootView (created by App)
    in Provider (created by App)
    in Theme (created by App)
    in PortalProviderComponent
    in Theme (created by ThemeProvider)
    in ThemeProvider (created by TamaguiProvider)
    in Provider (created by TamaguiProvider)
    in TamaguiProvider
    in TamaguiProvider (created by App)
    in App (created by withDevTools(App))
    in withDevTools(App)
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent), js engine: hermes`

**Comment by vilenleung** [comment]
i just rollback version from ^1.99.1 to ^1.95.3
it still happen and now i can not build and debug my project.

**Comment by SamuraiF0x** [comment]
Hey guys, I've tested this on both react native and expo and they both work for me. 
Try to delete `node_modules` and lock file, then install again using `yarn install`.

@ramspeedy, if you can give me write access I can push the `yarn.lock` that worked for me

**Comment by ramspeedy** [comment]
@SamuraiF0x sent a collaborator invite for the repo

**Comment by SamuraiF0x** [comment]
pushed the yarn.lock!

**Comment by rk-kc** [comment]
Thanks all. `yarn install` worked for me too.

**Comment by Phoenix654654** [comment]
> Hey guys, I've tested this on both react native and expo and they both work for me. Try to delete `node_modules` and lock file, then install again using `yarn install`.
> 
> @ramspeedy, if you can give me write access I can push the `yarn.lock` that worked for me


it didn't help me. i use expo

**Comment by Phoenix654654** [comment]
TextArea also show this error "TypeError: Cannot read property 'useRef' of null"

**Comment by SamuraiF0x** [comment]
@Phoenix654654 delete your node modules and lock file, copy yarn.lock I pushed to this repo https://github.com/ramspeedy/tamagui-test/tree/main and then yarn install

**Comment by joshbuckner** [comment]
I was receiving a number of similar errors from various components like `Select` and `Slider`.

I'm running an expo project with npm as the package manager. After switching my package manager from npm => yarn everything works.

**Comment by natew** [comment]
Can't repro anymore, lmk if an issue still

**Comment by mosleim** [comment]
if anyone got lost on this page, maybe you need to do the following:

1. make sure tamagui and all @tamagui/library have the same version
2. delete node_modules and lockfile (yarn-lock, pnpm-lock etc)
3. reinstall.

hope you guys can continue coding again. hehe
